src/main/kotlin/com/demonwav/mcdev/platform/mixin/inspection/MixinClassReferenceInspection.kt

```diff
@@ -41,7 +41,7 @@ class MixinClassReferenceInspection : MixinInspection() {
                 return
             }
 
-            holder.registerProblem(type, "${referencedClass.name} is a Mixin class, and cannot be referenced directly")
+            holder.registerProblem(type, "Mixin class cannot be referenced directly")
```

Comment:
Not sure if this is already handled elsewhere, but certain mixin classes *can* be referenced directly. Specifically *Accessor Mixins*.